packages/augur-ui/src/modules/contracts/actions/contractCalls.ts

```diff
@@ -279,7 +314,7 @@ export async function uniswapRepForEthRate(rep: BigNumber): Promise<BigNumber> {
 }
 
 export async function uniswapEthForDaiRate(wei: BigNumber): Promise<BigNumber> {
-  return new BigNumber(182);
+  return new BigNumber(148);
```

Comment:
This looks a little weird. Why 148

Reply:
we are using a mock exchange rate now (until Uniswap is ready). That was the price of ETH when i started this work. The previous mock value of 182 added some uneeded confusion to testing.